NAS2D/Xml/XmlElement.cpp

```diff
@@ -246,6 +246,20 @@ XmlNode* XmlElement::clone() const
 	return clone;
 }
 
+std::string XmlElement::GetText() const
```

Comment:
I believe the naming convention in this project is lowerCamelCase for function names.